contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogList.vue

```diff
@@ -46,60 +46,55 @@
             <ChannelItem
               :channelId="item.id"
               :detailsRouteName="detailsRouteName"
-              style="flex-grow: 1; width: 100%;"
+              style="flex-grow: 1; width: 100%"
             />
           </VLayout>
         </VFlex>
-        <VFlex xs12 style="padding-bottom: 72px;">
+        <VFlex xs12 style="padding-bottom: 72px">
           <VLayout justify-center>
-            <Pagination
-              :pageNumber="page.page_number"
-              :totalPages="page.total_pages"
-            />
+            <Pagination :pageNumber="page.page_number" :totalPages="page.total_pages" />
           </VLayout>
         </VFlex>
       </VLayout>
-      <BottomToolBar
+      <BottomBar
         v-if="selecting"
-        clipped-left
-        color="white"
-        flat
         data-test="toolbar"
         :height="$vuetify.breakpoint.xsOnly ? '72px' : '56px'"
       >
-        <VLayout row wrap align-center>
-          <VFlex xs12 sm4 class="pb-1">
-            {{ $tr('channelSelectionCount', { count: selectedCount }) }}
-          </VFlex>
-          <VFlex xs12 sm8>
-            <VLayout row>
-              <VSpacer />
-              <VBtn flat data-test="cancel" class="ma-0" @click="setSelection(false)">
-                {{ $tr('cancelButton') }}
-              </VBtn>
-              <Menu top>
-                <template #activator="{ on }">
-                  <VBtn color="primary" class="ma-0 mx-2" v-on="on">
-                    {{ $tr('downloadButton') }}
-                    <Icon class="ml-1">
-                      arrow_drop_up
-                    </Icon>
-                  </VBtn>
-                </template>
-                <VList>
-                  <VListTile @click="downloadPDF">
-                    <VListTileTitle>{{ $tr('downloadPDF') }}</VListTileTitle>
-                  </VListTile>
-                  <VListTile data-test="download-csv" @click="downloadCSV">
-                    <VListTileTitle>{{ $tr('downloadCSV') }}</VListTileTitle>
-                  </VListTile>
-                </VList>
-              </Menu>
-            </VLayout>
-          </VFlex>
-        </VLayout>
-
-      </BottomToolBar>
+        <div class="content-wrapper">
```

Comment:
Why do we need to add these wrappers? I tested it removing the wrapper and it seems to be working fine. Is there any particular reason for that?

Reply:
You are right, this wrapper is not required. I wrote this before setting BottomBar's display to flex. After setting it to flex, I forgot about them. I have removed these.